exercises/series/canonical-data.json

```diff
@@ -0,0 +1,73 @@
+{
+    "exercise": "series",
+    "version": "1.0.0",
+    "cases": [
+        {
+            "description": "slices of one",
+            "property": "slices",
+            "series": "01234",
+            "slice_length": 1,
```

Comment:
We are in the process of enforcing that all keys should be `lowerCamelCase` (see #987). Could you rename this property to `sliceLength` (and also in the other cases)?